src/components/Section.tsx

```diff
@@ -4,7 +4,7 @@ import { useExcalidrawContainer } from "./App";
 
 interface SectionProps extends React.HTMLProps<HTMLElement> {
   heading: string;
-  children: React.ReactNode | ((header: React.ReactNode) => React.ReactNode);
+  children?: any;
```

Comment:
This I feel is broken with React 18 types, @dwelle any better suggestion for this ?

Reply:
let's just use `React.FC` here. Pushed.